src/main/java/ui/CreateRecipeUI.java

```diff
@@ -97,21 +101,26 @@ public void actionPerformed(ActionEvent e) {
         this.setDefaultCloseOperation(JFrame.DISPOSE_ON_CLOSE);
         this.pack();
     }
-
+    @Override
     public void success(){
         JOptionPane.showMessageDialog(null, "Success");
         this.setVisible(false);
     }
-
-    public void failure(){
+    @Override
+    public void fail(){
         JOptionPane.showMessageDialog(null,
                 "Sorry, the recipe's name already exists.");
```

Comment:
Totally optional, but maybe a quick message saying what the user can do next? ie. "try another name"